src/condor_utils/file_transfer.h

```diff
@@ -538,10 +538,10 @@ class FileTransfer final: public Service {
 		// iwd       - relative paths are relative to this path
 		// max_depth - how deep to recurse (-1 for infinite)
 		// expanded_list - the list of files to transfer
-	static bool ExpandFileTransferList( char const *src_path, char const *dest_dir, char const *iwd, int max_depth, FileTransferList &expanded_list, bool preserveRelativePaths );
+	static bool ExpandFileTransferList( char const *src_path, char const *dest_dir, char const *iwd, int max_depth, FileTransferList &expanded_list, bool preserveRelativePaths, char const *SpoolSpace );
```

Comment:
The function is complex, subtle, and its desired functionality is nearly completely undocumented.

Since this takes a complex 6-argument function and turns it into a complex 7-argument function, could we spend some time and drop knowledge about what the desired behavior is (at least with respect to `SpoolSpace`; preferably more).

Reply:
New comment looks good, thanks.